CouchbaseLite.xcodeproj/project.pbxproj

```diff
@@ -3517,7 +3517,6 @@
 					};
 					27452BB71AB1180800327776 = {
 						CreatedOnToolsVersion = 6.3;
-						DevelopmentTeam = N2Q372V7W2;
```

Comment:
Yeah, it was meant to be there. If the dev team is specifically set to your Couchbase team it seems to cause issues.